source/Calamari/Calamari.csproj

```diff
@@ -25,6 +25,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Polly" Version="5.6.1" />
```

Comment:
This was added in order to get some new extension methods in retry handling so we can set a specific timeout if the response from Azure is a 429. 

The latest version is in the 7.x range, however it doesn't appear like we can use that as Calamari.Common is already using 5.4.0 and it causes issues if we move to 7 because certain assembly versions are missing. This showed up in the tests from the packaged Calamari executable and doesn't seem to happen using 5.6.1 so I think this is ok here.